images/linux/scripts/installers/1604/powershellcore.sh

```diff
@@ -8,6 +8,7 @@
 source $HELPER_SCRIPTS/document.sh
 
 LSB_RELEASE=$(lsb_release -rs)
+curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list > /etc/apt/sources.list.d/microsoft-prod.list
```

Comment:
The same repo is  installed in the repos.sh  script. Why do you put it again?

Reply:
string  has been removed.
commit: andy-mishechkin@aca1c5a

Reply:
this string has been removed:
https://github.com/andy-mishechkin/virtual-environments/commit/cb6d440538821c1bfa1876bf193e3f2d9d297dcb